apps/extension/webpack/webpack.dev.js

```diff
@@ -1,4 +1,4 @@
-import merge from 'webpack-merge';
+import { merge } from 'webpack-merge';
```

Comment:
This file was breaking without this change, which is weird, since it appears that `merge` is _also_ the default export of `webpack-merge`. But nonetheless, this change fixed it, so 🤷🏻‍♂️